src/components/10-atoms/toggle-switch/README.md

```diff
@@ -55,13 +55,13 @@ Import the toggle-switch-defining script and use a toggle-switch like this:
 
 ## Properties
 
-### on
+### active
 
-The boolean `on` sets the on/off visual state.
+The Boolean attribute `active` sets the on/off visual state.
```

Comment:
It not just sets the `visual` state. Also the "real" state of the internal checkbox. So just remove the word "visual". Or just "....sets the state". but like u want....